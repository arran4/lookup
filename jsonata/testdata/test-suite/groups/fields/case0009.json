{
  "dataset": "dataset1__INPUT",
  "bindings": {},
  "exprFile": "case0009.JSONATA"
}
