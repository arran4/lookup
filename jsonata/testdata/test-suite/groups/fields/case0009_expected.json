["0203 544 1234"]
